### ISSUE TITLE
Implement MLP

### ISSUE THREAD

**Issue by MichelMajdalani** [MichelMajdalani/comp551-miniproject3#1]
Multilayer perceptron: implement this from scratch based on the code available in the slides. Your implementation should include the backpropagation and the mini-batch gradient descent algorithm used (e.g., SGD). You are encouraged to change the activation function (e.g., use ReLU), and increase the number of layers, and play with the number of units per layer.

- [ ] Required implementation

- [ ] Extra creativity
